moveit_kinematics/kdl_kinematics_plugin/include/moveit/kdl_kinematics_plugin/chainiksolver_vel_pinv_mimic.hpp

```diff
@@ -51,21 +51,15 @@ class ChainIkSolverVel_pinv_mimic : public ChainIkSolverVel
   /**
    * Constructor of the solver
    *
-   * @param chain the chain to calculate the inverse velocity
-   * kinematics for
-   * @param num_mimic_joints The number of joints that are setup to
-   * follow other joints
-   * @param num_redundant_joints The number of redundant dofs
-   * @param position_ik false if you want to solve for the full 6 dof
-   * end-effector pose, true if you want to solve only for the 3 dof
-   * end-effector position.
-   * @param eps if a singular value is below this value, its
-   * inverse is set to zero, default: 0.00001
-   * @param maxiter maximum iterations for the svd calculation,
-   * default: 150
+   * @param chain the chain to calculate the inverse velocity kinematics for
+   * @param num_mimic_joints The number of joints that are setup to follow other joints
+   * @param position_ik false if you want to solve for the full 6 dof end-effector pose,
+   *        true if you want to solve only for the 3 dof end-effector position.
+   * @param eps if a singular value is below this value, its inverse is set to zero, default: 0.00001
+   * @param maxiter maximum iterations for the svd calculation, default: 150
    */
-  explicit ChainIkSolverVel_pinv_mimic(const Chain& chain, int num_mimic_joints = 0, int num_redundant_joints = 0,
-                                       bool position_ik = false, double eps = 0.00001, int maxiter = 150);
+  explicit ChainIkSolverVel_pinv_mimic(const Chain& chain, int num_mimic_joints = 0, bool position_ik = false,
```

Comment:
I know this isn't from this PR but why does this method name mix styles?

Reply:
I don't know either. Which name do you suggest?

Reply:
I renamed it to `ChainIkSolverVelMimicSVD` now.